### ISSUE TITLE
Schedule: VoiceOver doesn't announce day selected

### ISSUE THREAD

**Issue by astralbodies** [ThatConference/mobile-ios#1]
When you use VoiceOver to change the selected day (with the buttons left or right) VoiceOver announces the newly selected button label instead of the current selected day. This ends up telling the user the wrong day for the schedule currently on the screen. 

Steps to reproduce:
1. Turn on VoiceOver.
2. Select the schedule. August 7 should be selected by default.
3. Navigate to the button labeled August 8th. Select it.
4. VoiceOver announces August 9th (the label of the right button) rather than August 8th.
